games/src/main/scala/com/gu/game_review_parser/parsers/StandardParser.scala

```diff
@@ -0,0 +1,117 @@
+package com.gu.game_review_parser.parsers
+
+import com.gu.contentapi.client.model.v1.Content
+import com.gu.game_review_parser.ParsedGameReview
+import org.jsoup.Jsoup
+
+import scala.util.Try
+
+object StandardParser {
+  def parseContent(content: Content): Option[ParsedGameReview] = {
+    val parsed = for {
+      title <- getTitle(content.webTitle)
+      fields <- content.fields
+      body <- fields.body
+      details <- getDetails(body)
```

Comment:
Yeah, in the thrift model i've made them mostly optional, but for this 'standard' format they should all be present. We can look into parsing the less reliable reviews later on